pkg/registry/images.go

```diff
@@ -46,30 +46,52 @@ func (e AppRollbackError) Error() string {
 	return fmt.Sprintf("app:%s, version:%d", e.AppID, e.Sequence)
 }
 
+func shouldGarbageCollectImages(isKurl bool, kurlRegistryHost string, installParams kotsutil.InstallationParams, registrySettings types.RegistrySettings) bool {
+	if !installParams.EnableImageDeletion {
+		logger.Info("ignoring image garbage collection because image deletion is disabled")
```

Comment:
ideally, this function would return a "reason" why images shouldn't be garbage collected, and the caller function would decide to log that reason or not if that makes sense.